apis/infix-en_GB/atrium-api-infix-en_GB-common/src/main/kotlin/ch/tutteli/atrium/api/infix/en_GB/mapEntryExpectations.kt

```diff
@@ -13,3 +13,9 @@ import ch.tutteli.atrium.logic.*
  */
 infix fun <K, V, T : Map.Entry<K, V>> Expect<T>.toEqualKeyValue(keyValuePair: Pair<K, V>): Expect<T> =
     _logicAppend { isKeyValue(keyValuePair.first, keyValuePair.second) }
+
+infix fun <K, T : Map.Entry<K, *>> Expect<T>.toEqualKey(key: K): Expect<T> =
+    _logicAppend { isKey(key) }
+
+infix fun <V, T : Map.Entry<*, V>> Expect<T>.toEqualValue(value: V): Expect<T> =
+    _logicAppend { isValue(value) }
```

Comment:
please remove again. I appreciate your effort but don't think it is something which is really used that often that it deserves a shortcut. One can already write `key toEqual(...)` and `value toEqual(...)`